Tasks/Common/SonarQubeHelpers/SonarQubeHelper.ps1

```diff
@@ -12,22 +12,30 @@ function EscapeArg
 
 
 # Set a variable in a property bag that is accessible by all steps
-# To retrieve the variable use $val = Get-Variable $distributedTaskContext "varName"
+# To retrieve the variable use GetTaskContextVariable
+# The variable will be available in both the current and the subsequent tasks
 function SetTaskContextVariable
```

Comment:
Not familiar with this approach, and the documentation doesn't explain why you set the environment variable and call Set-VstsTaskVariable, so worth documenting that. So, why do we need them both?